common-component/src/main/java/shop/wazard/util/calculator/Calculator.java

```diff
@@ -12,7 +12,7 @@ public static double getAttitudeScore(int tardyCount, int absentCount, int total
     }
 
     private static double calculateAttitudeScore(int tardyCount, int absentCount, int totalWorkDay) {
-        double rawResult = 10 - (10 * (calculateTardyScore(tardyCount) + calculateAbsentScore(absentCount)) / (double) totalWorkDay);
+        double rawResult = 10 - (10 * (calculateTardyScore(tardyCount) + calculateAbsentScore(absentCount)) / (double) (totalWorkDay + absentCount));
```

Comment:
```suggestion
        double rawResult = 10 - (10 * (calculateTardyScore(tardyCount) + calculateAbsentScore(absentCount)) / (totalWorkDay + absentCount));
```
분자가 double이라 계산결과 묵시적 변환 되지 않나?

Reply:
총 근무 일 수 계산하는 메서드도 만들면 좋을 듯
```suggestion
        double penaltyPoints = calculatePenaltyPoints(tardyCount, absentCount, totalWork)
        double rawResult = 10 * (1 - penaltyPoints)
        return rawResult;
        
        
        private double calculatePenaltyPoints(int tardyCount, int absentCount, int totalWork) {
            return calculateTardyScore(tardyCount) + calculateAbsentScore(absentCount) / calculateTotalWorkDay(totalWork, absentCount));
        }
```

Reply:
메서드 더 분리 시켜버리는건 어때

Reply:
ㅇㅋㅇㅋ 다 바꿔볼게

Reply:
수정완료